src/blob/blob_client_wrapper.cpp

```diff
@@ -429,7 +433,7 @@ namespace azure {  namespace storage_lite {
             }
             catch(std::exception& ex)
             {
-                logger::log(log_level::error, "Unknown failure in upload_block_blob_from_stream.  ex.what() = %s, container = %s, blob = %s.", ex.what(), container.c_str(), blob.c_str());
+                logger::log(log_level::error, "Unknown failure in upload_block_blob_from_stream.  ex.what() = %s, container = %s, blob = %s Streamlen=%zu.", ex.what(), container.c_str(), blob.c_str(), streamlen);
```

Comment:
"Streamlen" it is better to align with "streamlen"

Reply:
Will do.